src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the beginning of the class definition.